static/less/group-detail.less

```diff
@@ -549,7 +549,6 @@ div.traceback > ul {
 
     &.collapsed {
       .title {
-        padding: 4px 20px 4px 20px;
```

Comment:
In non-native stacktraces, leads-to-app hints are displayed inline, whereas native stacktrace hints are displayed above the package name.  When context is expanded, this hint is removed. In the case of native stacktraces, this will shrink the row height since the text above is removed. In non-native stacktraces where the hint is inline, the row hight should not shrink. The forced padding above is therefore unnecessary.

Reply:
Thank you for the thorough explanation! This is very helpful 💯